### DISCUSSION TITLE
NET 8.0 Support?

### DISCUSSION THREAD

**Discussion by vivekN7** [Caliburn-Micro/Caliburn.Micro#893] [Q&A] [answered]
Hi! Firstly thank you to the team for helping to still maintain CM.
We have made some business applications heavily using Caliburn Micro (WPF) and are on .NET 6.0 right now.
Now that the next long term support .NET 8.0 is out, is there any plans for CM to be updated to support .NET 8 as well?
If not, we need to think about changing a lot of our codebase to another framework, and that would be very unfortunate since CM is fantastic.
Thanks!

**Comment by Mrxx99** [comment]
You should be able to just use it, there is no problem if a library targets a older framework, the same as .NET standard 2.0 libraries can still be used. They even benefit from .NET 8 performance improvements. There is no need to target newer versions unless the library wants to use newly introduced APIs.

And there is definitely no reason to swap out a library only if it does not target the latest framework version immediately.

**Reply by vivekN7** [reply]
Thank you. We will keep CM then.

**Comment by vb2ae** [comment, marked as answer]
There will be a .net 8 version soon but as @Mrxx99 states you can still use the .net 6 version

**Reply by vivekN7** [reply]
Thank you! That is very good to know.

**Comment by wstaelens** [comment]
we target .net 9, I assume this is not a problem with CM?

**Comment by vb2ae** [comment]
The latest beta supports .net 9  https://www.nuget.org/packages/Caliburn.Micro/5.0.181-beta#supportedframeworks-body-tab
